bridges/TwitterBridge.php

```diff
@@ -234,7 +234,31 @@ public function collectData(){
 
 			// generate the title
 			$item['title'] = $tweet->full_text;
-			$cleanedTweet  = $tweet->full_text;
+
+			// Convert plain text URLs into HTML hyperlinks
+			if (isset($tweet->entities->media)) {
+				foreach($tweet->entities->media as $media) {
+					$cleanedTweet = str_replace($media->url,
+						'<a href="' . $media->expanded_url . '">' . $media->display_url . '</a>',
+						$tweet->full_text);
+				}
+			} elseif (isset($tweet->entities->urls)) {
```

Comment:
If both media and urls are given, they should be converted.
All other "t.co" links, should be converted using regex'es.

I would suggest to use "if" instead of "elseif"

Reply:
Sounds good to me.